src/neural/network_tf_cc.cc

```diff
@@ -132,18 +138,49 @@ std::pair<Output, Output> MakeNetwork(const Scope& scope, Input input,
   }
 
   // Policy head
-  auto conv_pol =
-      MakeConvBlock<CPU>(scope, flow, 1, filters, 32, weights.policy);
-  if (CPU) {
-    // conv_pol = Transpose(scope, conv_pol, {0, 3, 1, 2});
+  Output policy_head;
+  if (!weights.policy1.weights.empty()) {
+    // Conv policy head.
+    auto conv_pol1 =
+        MakeConvBlock<CPU>(scope, flow, 3, filters, filters, weights.policy1);
+    auto conv_pol =
+        MakeConvBlock<CPU>(scope, conv_pol1, 3, filters, 80, weights.policy,
+                           nullptr, nullptr, /* relu= */ false);
+
+    // [1858 -> HWC or CHW]
+    std::vector<int> policy_map(1858);
+    for (const auto& mapping : kConvPolicyMap) {
+      if (mapping == -1) continue;
+      const auto index = &mapping - kConvPolicyMap;
+      const auto direction = index / 64;
```

Comment:
'direction' doesn't seem the most obvious name.  Its 73 options for different moves relative to square, for which there are only 16 distinct 'directions'.

Reply:
Changed to "displacement".

Reply:
better - we can think of it as a 3 dimensional displacement, 2d of movement and one of piece type change :P